### PR TITLE
refactor(measure): Update measure tool UI implementation.

### PR DESCRIPTION
**Pull request by bradh**
Replaces deprecated Action / ActionManager implementation.

### CODE REVIEW

File: src/os/ui/measurebutton.js [hunk 1]
```diff
@@ -163,7 +174,7 @@ os.ui.MeasureButtonCtrl.prototype.getMeasureInteraction_ = function() {
 
 
 /**
- * @param {os.ui.action.ActionEvent} evt The event from the menu
+ * @param {os.ui.menu.MenuEvent<Array<number>>} evt The event from the menu
```
Comment (wallw-bits) [inline src/os/ui/measurebutton.js]: Just for consistency, please make the handler type match the type provided by the menu. In this case, `os.ui.menu.MenuEvent<undefined>`.

Comment (schmidtk) [inline src/os/ui/measurebutton.js]: The generic type here should be `undefined` to match the menu type (describes the type of the menu context).

**Comment by bradh** [comment]
I see in #40 that the implementation was moved into `src/os/ui/menu/`. 

Should that change also be made here? That is, should `src/os/ui/measurebutton.js` be moved to be `src/os/ui/menu/measurebutton.js`?

**Comment by wallw-bits** [comment]
You can if you like but I don't really have an opinion on it either way. Our package structure is a gigantic mess due to some moves/merges of other projects in the past.

**Comment by bradh** [comment]
Incorporated change to type, did not move at this time. Force pushed an updated tree with the changes.

**Comment by schmidtk** [comment]
@bradh I moved the query area import menu because it was an isolated menu under `os.action` ("actions" are the old menu API). Yours is part of a controller/UI component so it doesn't need to move.

**Pull request merged**